DB/src/main/java/io/deephaven/db/v2/locations/local/ReadOnlyParquetTableLocation.java

```diff
@@ -192,6 +192,8 @@ private GroupingFile getGroupingFile(String name) {
                             }
                             break;
                         case INT96:
```

Comment:
Nope, this is Parquet 1.0, pre-logical types.  The new timestamp format is the one supported in logical types where you have a unit and a number of those since epoch.